src/WebDocs/wwwroot/app/functions/UnloadData/samples/001/content.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="/drapo.js"></script>
+    <title>Function - UnloadData</title>
+</head>
+<body>
+    <span>Function - UnloadData</span>
+    <div d-dataKey="data" d-dataUrlGet="/Data"></div>
```

Comment:
this API does not exist

Reply:
which api can i use?
'CreateNode' or add another one?

Reply:
You can use CreateNodes. You just need some data to unload it in the function